src/data/scout_census.py

```diff
@@ -44,25 +44,25 @@ class ScoutCensus:
         "sections": {
             "Beavers": {
                 "type": "Colony", "unit_label": "Beavers_Units", "level": "Group",
-                "male": "Beavers_m", "female": "Beavers_f",
+                "male": "Beavers_m", "female": "Beavers_f", "total": "Beavers_total",
```

Comment:
Do you mean like here?

Reply:
Or adding a new key:[ list ] pair with all 3?

Reply:
Objective is to remove the magic strings in this line, so that setup_data_file.py doesn't have to change if the census column name changes/
`scout_data.data[f"{section}_total"] = scout_data.data[[f"{section}_f", f"{section}_m", f"{section}_SelfIdentify", f"{section}_PreferNoToSay"]].sum(axis=1).astype("Int16")`
To become:
`scout_data.data[[section]["total"]] = scout_data.data[section["membership_columns"].sum(axis=1).astype("Int16")`
Meaning that scout_census.py is the only file that you would need to change if the column names in the census change.